src/main/flight/interpolated_setpoint.c

```diff
@@ -83,10 +82,10 @@ FAST_CODE_NOINLINE float interpolatedSpApply(int axis, bool newRcFrame, ffInterp
         prevRawSetpoint[axis] = rawSetpoint;
         
         if (axis == FD_ROLL) {
-            DEBUG_SET(DEBUG_FF_INTERPOLATED, 0, setpointDeltaImpl[axis] * 1000);
-            DEBUG_SET(DEBUG_FF_INTERPOLATED, 1, boostAmount * 1000);
-            DEBUG_SET(DEBUG_FF_INTERPOLATED, 2, boostAmount * clip * 1000);
-            DEBUG_SET(DEBUG_FF_INTERPOLATED, 3, setpointJerk * 1000);
+            DEBUG_SET(DEBUG_FF_INTERPOLATED, 0, setpointDeltaImpl[axis] * 100);
```

Comment:
This seems to be unrelated - can you please extract this into a separate pull request.

Reply:
I definitely can, though it was necessary to get the clip value in the debug to sort this out - without clip in the debug, the diagrams above can't be made.  

@joelucid set up the debugs originally.  Would it be OK to retain these changes here if he doesn't mind?  The clip percentage in a 0-100 value is easier to read, and the other values are easier to read in a 0-100 range also.  

However, if you'd prefer them in separate PR's to independently keep track of what was done and where, I'm happy to do that.

Reply:
The decision about being a 0-100 or 0-1000 range here must be related with the need of decimal places or not. If we need to show them in a 0-100 scale must be done in the Blackbox Explorer.

Reply:
@ctzsnooze: Yes please, separate them out. The debugging changes are a change of the API with blackbox lot viewer, and should go into the next release, and not be part of the bugfix that the rest of this is.

Reply:
What should I do about debug3, which was `setpointJerk * 1000`?  There no longer is any `setpointJerk` value, so I have to change that line.  Could I please leave that change and make debug3 into `clip * 1000`, since  `clip` is useful on its own.
The reason a scale of 100 is preferable for the `clip` value is that clip is easiest thought of as a percentage, clip in the code is a 0-1 value, and clip does not require decimal point precision for interpretation.  That's why I would prefer to keep `clip * 100` for debug3 if that was at all possible. 
The other values I don't care about much.  But they are already fairly large values.  Multiplying by 1000 makes them into unnecessarily large numbers.  That's why 100 is likely better.

Reply:
I have reverted debugs 0-2 to scale of 1000, and left debug3 as clip * 100, since that is the best and most useful purpose that debug3 can be used for now, considering that its original debug parameter no longer exists.